tools/memory_estimator/json_report_generator.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+from operator import truediv
+
+
+# Some of the file names are customized when diplaying on the website. This dict
+# contains the mappings of original file name to customized name with additional
+# information.
+__FILE_NAME_TO_DISPLAY_NAME__ = {
+        "http_parser.c": "http_parser.c (third-party utility)"
+    }
+
+
+def convert_size_to_kb(byte_size):
+    kb_size = round(truediv(int(byte_size),1024),1)
+    if (kb_size == 0.0):
+        return 0.1
+    else:
+        return kb_size
+
+
+def parse_sizes(sizes):
+    '''
+    It assumes the Berkley output format:
+    text       data     bss     dec     hex filename
+
+    Returns a dict of the following format:
+    {
+        'file_name':'size',
+        'file_name':'size'
+    }
+    '''
+    filename_to_size_dict = {}
+
+    for size in sizes[2:]:
```

Comment:
It would be nice to comment why we start from index 2. Maybe the filtering can be done outside the function?

Reply:
Added comment.